data-model/src/main/scala/za/co/absa/enceladus/model/UsedIn.scala

```diff
@@ -15,15 +15,42 @@
 
 package za.co.absa.enceladus.model
 
+import com.fasterxml.jackson.annotation.JsonIgnore
 import za.co.absa.enceladus.model.menas.MenasReference
 
-case class UsedIn(
-  datasets: Option[Seq[MenasReference]] = None,
-  mappingTables: Option[Seq[MenasReference]] = None
-) {
+case class UsedIn(datasets: Option[Seq[MenasReference]] = None,
+                  mappingTables: Option[Seq[MenasReference]] = None) {
 
-  def nonEmpty: Boolean = {
```

Comment:
In order to assess correctness of the rewrite, the unit test for nonEmpty was written first and the implementation was changed after that.